### DISCUSSION TITLE
updated this , please check and reply.

### DISCUSSION THREAD

**Discussion by ghost** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#181] [Show and tell]
https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/discussions/179

**Comment by AnonymousWP** [comment]
This is not how you're supposed to attract attention on another discussion. If you posted a new reply in that discussion and we'd have seen it too. Refrain from opening such pretty useless new discussions.
